benchmark-scheduled-unet-misa.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Usage: PATH=/path/to/iree/build/tools:$PATH ./benchmark-unet.sh N
+
+set -xeu
+
+if (( $# != 1 && $# != 2 )); then
+  echo "usage: $0 <hip-device-id> [<ipra-path-prefix>]"
+  exit 1
+fi
+
+IRPA_PATH_PREFIX="${2:-/data/shark}"
+
+iree-benchmark-module \
+  --device=rocm://$1 \
+  --device_allocator=caching \
+  --module=$PWD/tmp/scheduled_unet_misa.vmfb \
```

Comment:
```suggestion
  --module="$PWD"/tmp/scheduled_unet_misa.vmfb \
```
quote to account for non-standard paths, e.g., with spaces